src/engine/client/cl_irc.cpp

```diff
@@ -1929,7 +1929,7 @@ static ircCmd_t CTCP_Version( bool is_channel, const char * )
 		return ircCmd_t::SUCCESS;
 	}
 
-	return IRC_Send( "NOTICE %s :\001VERSION Daemon IRC client — v\n" Q3_VERSION "\001", IRC_String( pfx_nickOrServer ) );
+	return IRC_Send( "NOTICE %s :\001VERSION Daemon IRC client — v\n" Q3_ENGINE "\001", IRC_String( pfx_nickOrServer ) );
```

Comment:
Does anyone know if the IRC code is supposed to work at all?

Reply:
IRC is now nuked: https://github.com/DaemonEngine/Daemon/commit/ab3f6591e3c9c42b0447c100427d28aba32e87c6